File: examples/flutter/lib/src/models/jwt.dart
part of appwrite.models;

/// JWT
class Jwt {
    /// JWT encoded string.
    final String jwt;

    Jwt({
        required this.jwt,
    });

    factory Jwt.fromMap(Map<String, dynamic> map) {
        return Jwt(
            jwt: map['jwt'],
        );
    }

    Map<String, dynamic> toMap() {
        return {
            "jwt": jwt,
        };
    }
}

File: examples/swift-client/Sources/AppwriteModels/Jwt.swift

/// JWT
public class Jwt {

    /// JWT encoded string.
    public let jwt: String

    init(
        jwt: String
    ) {
        self.jwt = jwt
    }

    public static func from(map: [String: Any]) -> Jwt {
        return Jwt(
            jwt: map["jwt"] as! String
        )
    }

    public func toMap() -> [String: Any] {
        return [
            "jwt": jwt as Any
        ]
    }
        
}